libs/payments/ui/src/lib/actions/setupCart.ts

```diff
@@ -15,9 +15,7 @@ export const setupCartAction = async (
   uid?: string,
   ip?: string
 ) => {
-  const actionsService = await app.getActionsService();
-
-  await actionsService.setupCart(
+  return app.getActionsService().setupCart(
```

Comment:
Doesn't `app.getActionsService()` return a promise?

Reply:
Not anymore. After Next 14.2 landed with the instrumentation fixes, the code was updated to boot up NestApp on startup. 

PR here #16704

Reply:
Sweet!